src/SFA.DAS.EAS.Portal/Application/Commands/AddReservationCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CosmosDb;
+using SFA.DAS.EAS.Portal.Database;
+using SFA.DAS.EAS.Portal.Database.Models;
+using SFA.DAS.Reservations.Messages;
+
+namespace SFA.DAS.EAS.Portal.Application.Commands
+{
+    public class AddReservationCommand
```

Comment:
Is this a command or command handler?  Should be separation of concerns as per the standard pattern command.handler pattern.

Reply:
It's a command object and there is no 'standard' pattern.

From wikipedia's [page](https://en.wikipedia.org/wiki/Command_pattern) on the command pattern:
'In some implementations the handler code is part of the command/action object. In other implementations the code is part of the Receiver/Target Object, and in yet other implementations the handler code is kept separate from the other objects.'

Reply:
As we've discussed, and as mentioned in another comment, we can decide on a common pattern for the handlers/commands, but we don't need to do that for this PR.